src/app/dashboard/shared/base-sensor/base-sensor.test.ts

```diff
@@ -36,29 +36,43 @@ describe('base-sensor', () => {
     });
 
     describe('when initialize component', () => {
-        beforeEach(() => {
+
+        it('should initialize sensor data with provided value', () => {
+            sut.device = { mqttId: 'For test', value: 'initvalue' };
             sut.ngOnInit();
+
+            expect(sut.data).toEqual({
+                value: 'initvalue',
+                updateTime: null
+            });
         });
 
         it('should initialize sensor data', () => {
+            sut.ngOnInit();
```

Comment:
It'd be better to add sut.noOnInit() to beforeEach instead of calling it in every test.